lexicons/app/bsky/feed/defs.json

```diff
@@ -31,7 +31,11 @@
           "type": "array",
           "items": { "type": "ref", "ref": "com.atproto.label.defs#label" }
         },
-        "threadgate": { "type": "ref", "ref": "#threadgateView" }
+        "threadgate": { "type": "ref", "ref": "#threadgateView" },
+        "tags": {
```

Comment:
just curious - is there a reason to elevate this to the postView instead of just getting the data from within the record (similar to how we handle text/facets/etc)?

i'm not opposed to this, but most other fields that we elevate to the view have some additional info/hydration that go along with them

Reply:
for instance, we don't have `langs` on `postView`

Reply:
Ah ok yeah, I suppose not 🤔 I'll remove this functionality

Reply:
we can always add it in later if we realize we need it - it's non-breaking